Wikipedia/Code/ReadingListsController.swift

```diff
@@ -1,16 +1,20 @@
 import Foundation
 
-internal let WMFReadingListSyncStateKey = "WMFReadingListsSyncState"
-internal let WMFReadingListDefaultListEnabledKey = "WMFReadingListDefaultListEnabled"
-fileprivate let WMFReadingListSyncRemotelyEnabledKey = "WMFReadingListSyncRemotelyEnabled"
+// Sync keys
+let WMFReadingListSyncStateKey = "WMFReadingListsSyncState"
+private let WMFReadingListSyncRemotelyEnabledKey = "WMFReadingListSyncRemotelyEnabled"
```

Comment:
This can remain `fileprivate`.

Reply:
private is more restrictive and it looks like we don't need to make it fileprivate in this case

Reply:
> private is more restrictive

whoa interesting swift 4 change! reading that now... (good catch!)